wolfcrypt/src/ecc.c

```diff
@@ -6300,8 +6301,8 @@ int wc_ecc_import_point_der(byte* in, word32 inLen, const int curve_idx,
     inLen -= 1;
     in += 1;
 
-    /* calculate key size based on inLen / 2 */
-    keysize = inLen>>1;
+    /* calculate key size based on inLen / 2 if uncompressed */
+    keysize = compressed ? inLen : inLen>>1;
```

Comment:
Good fix, but it will break things.

Reply:
What is the alternative?

Reply:
New API.
Need to have an extended API that does things properly with different behaviour.

Reply:
Leave it as it is.
It is used for compressed as it is.

Reply:
What should I call the new API?

Reply:
Yep!

Reply:
Created a new API and preserved the functionality of the old one.